R/pkg/DESCRIPTION

```diff
@@ -13,6 +13,7 @@ Authors@R: c(person("Shivaram", "Venkataraman", role = c("aut", "cre"),
 License: Apache License (== 2.0)
 URL: http://www.apache.org/ http://spark.apache.org/
 BugReports: http://spark.apache.org/contributing.html
+SystemRequirements: Java (== 8)
 Depends:
     R (>= 3.0),
```

Comment:
btw, I saw this the other day, and thought we should update this to `>= 3.11` to reflect what we test with?
what do you think?

Reply:
Thats a valid point. Lets discuss this in JIRA / mailing list ? I think if it still works for 3.0 we should keep it because R might throw an error / warning for users on 3.0.

Reply:
yap let's separate this out